README.md

```diff
@@ -16,8 +16,6 @@ It was built with immutability and declarative configuration as code in mind.
 
 Out of the box it provides:
 - integration with Kubernetes ([Jenkins kubernetes-plugin](https://github.com/jenkinsci/kubernetes-plugin))
-- pipelines as code ([Jenkins pipelines](https://jenkins.io/doc/book/pipeline/))
-- extensibility via groovy scripts (similar to [Jenkins script console](https://wiki.jenkins.io/display/JENKINS/Jenkins+Script+Console)) or ([configuration as code plugin](https://github.com/jenkinsci/configuration-as-code-plugin))
```

Comment:
I think this should stay too.

Reply:
I think this can be misleading based on what the operator provides and what is provided by Jenkins. I will update the PR appropriately.

Reply:
I have rewriteen this info considering that it is a preliminary to understand before consuming the operator.

Reply:
I would just change wording to something like:
`- preconfigured Configuration as Code plugin for extensibility, easy to use with operator.....`
or something along those lines

Reply:
bump